specs/rearrangements.yaml

```diff
@@ -139,6 +147,10 @@ fields:
     type: integer
     doc: FWR4 end coord in sequence (transferred from germline)
   -
+    name: duplicate_count
+    type: integer
+    doc: Count of duplicate sequences for this sequence
```

Comment:
k, that's what I thought...maybe change to "Count of duplicate reads for this sequence"?